fr/health/centreon-agent.md

```diff
@@ -0,0 +1,503 @@
+
+## Tester l'Agent
+
+### Tester le service centreon-agent
+
+À cette étape, le service **centreon-agent** doit tourner, et être paramétré pour se lancer au démarrage du système.
+La commande suivante vérifie que le service a été configuré correctement :
+
+```
+systemctl status centreon-agent
+```
+
+Si tout s'est bien passé, le résultat ressemblera à l'exemple suivant :
+
+```
+systemctl status centreon-agent
+● centreon-agent.service - The Centreon Agent collect metrics and send them to Centreon SaaS Platform
+   Loaded: loaded (/etc/systemd/system/centreon-agent.service; enabled; vendor preset: disabled)
+   Active: active (running) since ven. 2019-11-08 14:52:26 CET; 5 days ago
+ Main PID: 22331 (centreon-agent)
+   CGroup: /system.slice/centreon-agent.service
+           └─22331 /usr/sbin/centreon-agent run
+```
+
+### Tester la collecte de données
+
+Une fois l'installation et la configuration terminées, vous pouvez utiliser la commande suivante pour forcer la collecte de données :
+
+```
+centreon-agent sample
+```
+
+Le résultat devrait ressembler à ça :
+
+```
+1624977737000000// centreonengine_uptime_seconds{_cmaas=cmco,hostname=val-central.centreon.io,os=linux,osfamily=rhel} 693583
+1624977737000000// centreonengine_command_buffers_used{_cmaas=cmco,hostname=val-central.centreon.io,os=linux,osfamily=rhel} 0
+1624977737000000// centreonengine_command_buffers_high{_cmaas=cmco,hostname=val-central.centreon.io,os=linux,osfamily=rhel} 0
+1624977737000000// centreonengine_command_buffers_total{_cmaas=cmco,hostname=val-central.centreon.io,os=linux,osfamily=rhel} 4096
+1624977737000000// centreonengine_external_command_1m{_cmaas=cmco,hostname=val-central.centreon.io,os=linux,osfamily=rhel} 0
+1624977737000000// centreonengine_general_external_command_5m{_cmaas=cmco,hostname=val-central.centreon.io,os=linux,osfamily=rhel} 0
+```
+
+Si vous obtenez des erreurs en testant la collecte, les logs du fichier `/var/log/centreon-agent/centreon-agent.log` peuvent vous aider à résoudre le problème.
+
+### Tester l'accès à la Plateforme Centreon Cloud
+
+Une fois l'installation et la configuration terminées, utilisez la commande suivante pour tester la connexion entre l'Agent et la Plateforme Centreon Cloud :
+
+```
+centreon-agent ping --config [chemin vers votre fichier centreon-agent.yml]
+```
+
+L'Agent retournera l'un des messages suivants :
+
+- **Unable to reach the Centreon Cloud Platform, check your network configuration** : vérifiez votre configuration réseau (proxy etc.)
+
+- **Centreon Cloud Platform reached successfully but your token is not recognized** : votre jeton n'est pas reconnu
+
+- **Centreon Cloud Platform reached successfully and authentication was successful** : l'Agent est bien connecté à notre plateforme.
+
+## Mettre à jour l'Agent
+
+Pour mettre à jour l'Agent, entrez :
+
+```
+yum update centreon-agent
```

Comment:
It could be useful to clean all repository cache before updating. (like we do with centreon)